### PR TITLE
New Career Roadmap/Timeline Component

### PR DESCRIPTION
**Pull request by Uri-E**
Added new timeline component to the CDLS. First
draft is for review and to discuss how it will be
implemented in Cascade.

### CODE REVIEW

File: components/03-sections/12-timeline/timeline.readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Image Collage Documentation
```
Comment (lalov1) [inline components/03-sections/12-timeline/timeline.readme.md:1]: Change to Timeline Documentation

**Review by lalov1 — COMMENTED** [review]
Minor changes to the documentation, but everything else looks good.

**Review by lalov1 — APPROVED** [review]

File: components/03-sections/12-timeline/timeline.hbs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    <h2>{{ headline }}</h2>
+    <div class="main-timeline">
+      <div class="col-12 timeline-banner-start"><h3>{{ subhead }}</h3></div>
+      <div class="timeline timeline-right">
+        <div class="card">
+          <div class="card-body p-4">
+            <h3 class="timeline-section-header">1. Section Header</h3>
+            <div class="row">
+              <div class="col-xs-12 col-md-6">
+                  <ul>
```
Comment (prd409-utsa) [inline components/03-sections/12-timeline/timeline.hbs:12]: On small devices, the two **ul** will stack into one column. They have space between them unless you remove the bottom margin from the first **ul**.

Screen Shot 2024-10-30 at 6 36 28 PM

Comment (garza-utsa) [reply components/03-sections/12-timeline/timeline.hbs:12]: agreed, @Uri-E, can you adjust the spacing on mobile view for these stacking items?

**Review by garza-utsa — CHANGES_REQUESTED** [review]
stacking lists on mobile need spacing adjustments

**Review by prd409-utsa — APPROVED** [review]

**Pull request merged**